ext/zip/php_zip.stub.php

```diff
@@ -184,6 +186,15 @@ public function getFromName(string $name, int $len = 0, int $flags = 0): string|
     public function getFromIndex(int $index, int $len = 0, int $flags = 0): string|false {}
 
     /** @return resource|false */
+    public function getStreamIndex(int $index, int $flags = 0) {}
```

Comment:
I would have swore that this method returned an index, and the other one returned a string only by looking at the method name. Would you think using `getStreamByIndex()` and `getStreamByName()` clarified their behavior?

Reply:
Yeah, but that would be inconsistent with the other methods: https://www.php.net/ziparchive.

Reply:
Ah, that's quite unfortunate :/

Reply:
Without BC, I will probably have used a single function, checking type of the first arg  ;)

`public function getStream(string|integer $entry, flags=0);`